test/test_dt2js.js

```diff
@@ -9,10 +9,19 @@ var fs = require('fs')
 
 var RAML_FILE_NAME = join(__dirname, 'examples/types_example.raml')
 
+describe.only('dt2js.getRAMLContext()', function () {
+  var getRAMLContext = dt2js.__get__('getRAMLContext')
+  it('should load included json file', function () {
+    var ramlData = fs.readFileSync(join(__dirname, 'examples/type_include.raml')).toString()
```

Comment:
I would put this type with the other types used for unit testing, inside `types_example.raml` and rename that `included.json` file `included_example.json` to be consistent with the rest...